### PR TITLE
Made single-file deployments work again

### PR DESCRIPTION
**Pull request by leo**
The function is basically always receiving an array of files, even if it's just a single one that the user is trying to upload. In turn, we need to pick the first array item for single file uploads (and not pass the whole array...) 😄

### CODE REVIEW

File: main/utils/deploy/index.js [hunk 1]
```diff
@@ -241,7 +241,9 @@ class Now extends EventEmitter {
 
       if (sizeExceeded) {
         showError(
-          `${sizeExceeded} of the files exceeded the limit for your plan. Please upgrade`
+          `${
+            sizeExceeded
+          } of the files exceeded the limit for your plan. Please upgrade`
```
Comment (Qix-) [inline main/utils/deploy/index.js]: Is this prettier doing this?

Comment (timneutkens) [reply main/utils/deploy/index.js]: Most likely

Comment (leo) [reply main/utils/deploy/index.js]: Yep!

Comment (Qix-) [reply main/utils/deploy/index.js]: Right I understand that but the syntax is nonsensical. There's absolutely no readability benefit for Prettier to be formatting code like this.

**Pull request merged**